Kitodo-DataFormat/src/test/resources/meta_missing_createdate.xml

```diff
@@ -0,0 +1,2760 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please use a shorter file for test cases which only represents neccessary tags.

Reply:
File is shorted.